api/tests/acceptance/application/organization-controller_test.js

```diff
@@ -1024,21 +1026,18 @@ describe('Acceptance | Application | organization-controller', () => {
         organizationRole: Membership.roles.ADMIN,
       });
 
-      databaseBuilder.factory.buildOrganizationInvitation({
+      firstOrganizationInvitation = databaseBuilder.factory.buildOrganizationInvitation({
         organizationId: organization.id,
```

Comment:
Si dans le beforeEach on fait directement
`organizationId = databaseBuilder.factory.buildOrganization().id;`

Alors on peut écrire ici
```suggestion
        organizationId,
```

Tout ce describe n'utilise que l'id de l'organisation

Reply:
Done